src/main/java/at/porscheinformatik/seleniumcomponents/clarity/ClarityIconComponent.java

```diff
@@ -1,24 +1,34 @@
 package at.porscheinformatik.seleniumcomponents.clarity;
 
+import static at.porscheinformatik.seleniumcomponents.WebElementSelector.*;
+
 import at.porscheinformatik.seleniumcomponents.AbstractSeleniumComponent;
 import at.porscheinformatik.seleniumcomponents.SeleniumComponent;
 import at.porscheinformatik.seleniumcomponents.WebElementSelector;
 
 public class ClarityIconComponent extends AbstractSeleniumComponent
 {
-    public static ClarityIconComponent in(SeleniumComponent parent)
```

Comment:
I forgot other teams may use this library too. I'll add it back, though I personally don't consider it very useful. Often there are a lot of icons within a component, so this would only get the first one found.